jwst/cal_ver_steps.py

```diff
@@ -43,7 +42,7 @@ def __init__(self, author, **pars):
         """
         # Initialize output reference file information based on user input
         descrip = pars.get('pars', "JWST calibration processing step version reference file")
-        history = pars.get('history', "Created by cal_ver_steps version {}".format(__version__))
+        history = pars.get('history', "Created by jwst version {}".format(__version__))
```

Comment:
How much does this change matter? Will this break something elsewhere?

Reply:
No we aren't using it anymore, at least to my knowledge. It was used in the past to create one giant list of the versions of all the sub-packages, which was then stored in a file. I don't know if @jaytmiller is still using that file anywhere in CRDS, but we don't use it for the CAL pipeline anymore.